connectors-element-template-links.txt

```diff
@@ -0,0 +1,134 @@
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/rest
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/salesforce
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/sendgrid
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/slack
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/slack-inbound
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/slack/?slack=inbound
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/slack/?slack=outbound
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/twilio
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/twilio/
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/twilio/?twilio=inbound
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/twilio/?twilio=outbound
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/twilio-webhook
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/uipath
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/whatsapp
+https://docs.camunda.io/docs/components/connectors/protocol/amazon-eventbridge/?awseventbridge=inbound
+https://docs.camunda.io/docs/components/connectors/protocol/amazon-eventbridge/?awseventbridge=outbound
+https://docs.camunda.io/docs/components/connectors/protocol/aws-lambda/#aws-authentication-types
+https://docs.camunda.io/docs/components/connectors/protocol/graphql
+https://docs.camunda.io/docs/components/connectors/protocol/graphql/#querymutation
+https://docs.camunda.io/docs/components/connectors/protocol/http-webhook
+https://docs.camunda.io/docs/components/connectors/protocol/polling
+https://docs.camunda.io/docs/components/connectors/protocol/rest
+https://docs.camunda.io/docs/components/connectors/protocol/soap
+https://docs.camunda.io/docs/components/connectors/use-connectors/
+https://docs.camunda.io/docs/components/connectors/use-connectors/#bpmn-errors
+https://docs.camunda.io/docs/components/connectors/use-connectors/#result-expression
+https://docs.camunda.io/docs/components/connectors/use-connectors/#result-variable
+https://docs.camunda.io/docs/components/modeler/bpmn/none-events
+https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/using-templates/#replacing-templates
+https://docs.camunda.io/docs/components/modeler/web-modeler/camunda-marketplace
+https://docs.camunda.io/docs/components/modeler/web-modeler/connectors/available-connectors/automation-anywhere/
+https://docs.camunda.io/docs/components/modeler/web-modeler/connectors/available-connectors/rest
+https://docs.camunda.io/docs/components/modeler/web-modeler/connectors/available-connectors/sendgrid
```

Comment:
@ztefanie these URLs are a little surprising. Were connectors ever nested under the modeler docs? 

The answer to that question is probably not important, but adding redirects for these URLs made me wonder if these are URLs that can be changed at the source, or if there's anything else funny about them.

Reply:
Good point. I just checked. These links where added to our codebase 3 years ago. I will adjust them for newer element templates, but cannot change them for older ones, therefore we sill need redirects for them.